File: app/src/main/java/dev/leonlatsch/kolibri/util/Generator.java
package dev.leonlatsch.kolibri.util;

import java.util.UUID;

/**
 * @author Leon Latsch
 * @since 1.0.0
 */
public class Generator {

    public static String genUUid() {
        UUID uuid = UUID.randomUUID();
        return uuid.toString();
    }
}

File: app/src/main/kotlin/dev/leonlatsch/kolibri/util/Generator.kt
package dev.leonlatsch.kolibri.util

import java.util.*

/**
 * @author Leon Latsch
 * @since 1.0.0
 */
object Generator {

    fun genUUid(): String {
        val uuid = UUID.randomUUID()
        return uuid.toString()
    }
}
